src/style.css

```diff
@@ -1 +1,30 @@
-/* add styles here */
+.page {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars__sta:last-child {
```

Comment:
There is a typo in the selector '.stars__sta:last-child'. It should be '.stars__star:last-child' to correctly apply the style to the last star element.